.circleci/workflows.yml

```diff
@@ -205,7 +205,7 @@ jobs:
       - invoke-pester:
           JumpCloudApiKey: $env:XAPIKEY_PESTER
           JumpCloudApiKeyMsp: $env:XAPIKEY_PESTER_MTP
-          ExcludeTagList: 'ModuleValidation, JCUsersFromCSV, JCDeployment'
+          ExcludeTagList: 'ModuleValidation, JCDeployment'
```

Comment:
We are not including the JCUsersFromCSV in our tests, these are working with some exceptions, see test CSV files for more info